x-pack/legacy/plugins/siem/server/lib/timeline/saved_object.ts

```diff
@@ -77,54 +78,68 @@ export class Timeline {
     request: FrameworkRequest,
     timelineId: string | null
   ): Promise<ResponseFavoriteTimeline> {
-    let timeline: SavedTimeline = {};
-    if (timelineId != null) {
-      const {
-        eventIdToNoteIds,
-        notes,
-        noteIds,
-        pinnedEventIds,
-        pinnedEventsSaveObject,
-        savedObjectId,
-        version,
-        ...savedTimeline
-      } = await this.getBasicSavedTimeline(request, timelineId);
-      timelineId = savedObjectId;
-      timeline = savedTimeline;
-    }
     const userName = getOr(null, 'credentials.username', request[internalFrameworkRequest].auth);
     const fullName = getOr(null, 'credentials.fullname', request[internalFrameworkRequest].auth);
-    const userFavoriteTimeline = {
-      keySearch: userName != null ? convertStringToBase64(userName) : null,
-      favoriteDate: new Date().valueOf(),
-      fullName,
-      userName,
-    };
-    if (timeline.favorite != null) {
-      const alreadyExistsTimelineFavoriteByUser = timeline.favorite.findIndex(
-        user => user.userName === userName
-      );
+    try {
+      let timeline: SavedTimeline = {};
+      if (timelineId != null) {
+        const {
+          eventIdToNoteIds,
+          notes,
+          noteIds,
+          pinnedEventIds,
+          pinnedEventsSaveObject,
+          savedObjectId,
+          version,
+          ...savedTimeline
```

Comment:
should this moving this up to line `87`?

Reply:
this was done intentionally to exclude attributes